flink-runtime/src/main/java/org/apache/flink/runtime/operators/DriverStrategy.java

```diff
@@ -73,6 +73,8 @@
 	LEFT_OUTER_MERGE(LeftOuterJoinDriver.class, null, MATERIALIZING, MATERIALIZING, 2),
 	RIGHT_OUTER_MERGE(RightOuterJoinDriver.class, null, MATERIALIZING, MATERIALIZING, 2),
 	FULL_OUTER_MERGE(FullOuterJoinDriver.class, null, MATERIALIZING, MATERIALIZING, 2),
+	FULL_OUTER_HYBRIDHASH_BUILD_FIRST(FullOuterJoinDriver.class, null, FULL_DAM, MATERIALIZING, 2),
```

Comment:
The strategies in this file are ordered by algorithm type, not by logical operator type. 
These hash join strategies should be put next to the other hash join strategies.